src/savegame_json.cpp

```diff
@@ -2733,3 +2734,78 @@ void deserialize( recipe_subset &value, JsonIn &jsin )
         value.include( &recipe_id( jsin.get_string() ).obj() );
     }
 }
+
+// basecamp
+void basecamp::serialize( JsonOut &json ) const
+{
+    json.start_object();
+    json.member( "name", name );
+    json.member( "x", pos.x );
```

Comment:
You can serialize `tripoint`s and `point`s directly.

Reply:
Nice! Thanks.